airbyte-integrations/connectors/source-slack/source_slack/spec.json

```diff
@@ -26,6 +26,13 @@
         "title": "Join all channels",
         "description": "Whether to join all channels or to sync data only from channels the bot is already in.  If false, you'll need to manually add the bot to all the channels from which you'd like to sync messages. "
       },
+      "channel_filter": {
+        "type": "string",
+        "default": "",
+        "title": "Channel name filter",
+        "description": "A comma-delimited channel names string (without leading '#' char) which limit the channels from which you'd like to sync. Empty string means no filter.",
+        "examples": ["channel_one,channel_two"]
+      },
```

Comment:
```suggestion
      "channel_filter": {
        "type": "array",
        "default": [],
        "title": "Channel name filter",
        "description": "A comma-delimited channel names string (without leading '#' char) which limit the channels from which you'd like to sync. Empty string means no filter.",
        "examples": ["channel_one,channel_two"]
      },
```

Reply:
Are you sure not using string? The filter is comma-delimited string specified by user and it will be split to a channel name list.

Reply:
alright, I just saw some comments below so I've changed it to list.